docs/reference/rest-api/root.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[rest-api-root]]
+=== Root API
```

Comment:
```suggestion
== Root API
```

Similar to `info.asciidoc`, this page is not a subpage, so all your headings need to be moved up one level.